pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -98,16 +98,16 @@ var (
 
 var (
 	// Variables associated with clear cache squashing.
-	clearCacheMutex sync.Mutex
+	clearCacheMutex sync.RWMutex
 
 	// lastClearCache is the time we last pushed
 	lastClearCache time.Time
 
 	// lastClearCacheEvent is the time of the last config event
 	lastClearCacheEvent time.Time
 
-	// clearCacheEvents is the counter of 'clearCache' calls
-	clearCacheEvents int
+	// clearCacheCounter is the counter of 'clearCache' calls
+	clearCacheCounter int
```

Comment:
why not stick with clearCacheEvents - seems more descriptive

Reply:
`clearCacheEvents ` makes me  confused, it is a counter.